EF_Example/Program.cs

```diff
@@ -12,7 +11,8 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
-builder.Services.AddDbContext<ExampleDbContext>(options=> options.UseSqlServer("Data Source=localhost;Initial Catalog=BookShop;user id=sa;password=test1234"));
+
+builder.Services.AddDbContext<ExampleDbContext>(options=> options.UseSqlServer(builder.Configuration.GetSection("AuthDataSettings")["ConnectionString"]));
```

Comment:
Looking at ioptions in the future might be nice but for now saves it being hard coded